packages/portfolio-contract/src/plan-solve.ts

```diff
@@ -87,6 +87,21 @@ export interface SolvedEdgeFlow {
 /** Model shape for javascript-lp-solver */
 export type LpModel = IModel<string, string>;
 
+/**
+ * Link to Factory and Wallet contracts:
+ * https://github.com/agoric-labs/agoric-to-axelar-local/blob/cd6087fa44de3b019b2cdac6962bb49b6a2bc1ca/packages/axelar-local-dev-cosmos/src/__tests__/contracts/Factory.sol
+ *
+ * Gas estimation interface - mirrors services/ymax-planner/src/gas-estimation.ts
+ * - getWalletEstimate: Estimate gas fees for remote wallet operations on the specified chain
+ * - getFactoryContractEstimate: Estimate gas fees for executing the factory contract on the specified chain
+ * - getReturnFeeEstimate: Estimate return fees for sending a transaction back from the factory contract to Agoric
```

Comment:
Suggested reformatting and quasi-chronological reördering:
```suggestion
 * Gas estimation interface -
 *   @see {@link ../../../services/ymax-planner/src/gas-estimation.ts}
 * - getFactoryContractEstimate: Estimate gas fees for executing the factory
 *   contract to create a wallet on the specified chain
 * - getReturnFeeEstimate: Estimate return fees for sending a transaction back
 *   from the factory contract to Agoric
 * - getWalletEstimate: Estimate gas fees for remote wallet operations on the
 *   specified chain
```